web/src/App.jsx

```diff
@@ -18,7 +18,7 @@ import PropTypes from "prop-types";
 export const usePersistentStore = create(
     persist(
         (set) => ({
-            selectedSource: 0,
+            selectedSource: null,
```

Comment:
why did you do this? if this is a bug, it's not explained.

0 is [falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy), and it appears you just moved the setting logic to something in PlayerCardFb

Reply:
This looks like I was attempting to do an early crack at what I ended up doing in #788, being that this handles autoselection and is in a branch that predates the successfully merged autoselection functionality